idowuadeleke/BANKA
# chore(Refactor):Refactor Codebase

### What does this PR do?
Refactor Codebase

### Description of Task to be completed?
- Replace login route to signin

### How should this be manually tested?
- clone the repo `git clone https://github.com/idowuadeleke/BANKA.git`

- on your terminal run `cd BANKA`

- run `git checkout ch-165444623`

- run `npm install`

- run `npm run start`

- make a request to `POST localhost:3000/api/v1/auth/signin`
  NB see screenshot below on the request data to pass

### Any background context you want to provide?
NA
### What are the relevant pivotal tracker stories?
[#165444623](https://www.pivotaltracker.com/story/show/165444623)

### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/40268693/56364745-3bcad900-61e7-11e9-83fd-8b69640ad88e.png)
